### PR TITLE
[Windows] Support Sysmon New EventIDs - 8, 9, 19, 20, 27, 28, 255

### PR DESCRIPTION
**Pull request by kcreddy**
## What does this PR do?
Supports more Sysmon events introduced since v13.10. Following EventIDs are supported:

EventIDs: `8, 9, 19, 20, 27, 28, 255`
 

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally
`elastic-package stack down && elastic-package build && elastic-package stack up --version=8.6.0 -d -v --services=elasticsearch && eval "$(elastic-package stack shellinit)" && elastic-package test pipeline --generate --data-streams sysmon_operational -v`

```
Run pipeline tests for the package
--- Test results for package: windows - START ---
╭─────────┬────────────────────┬───────────┬──────────────────┬────────┬──────────────╮
│ PACKAGE │ DATA STREAM        │ TEST TYPE │ TEST NAME        │ RESULT │ TIME ELAPSED │
├─────────┼────────────────────┼───────────┼──────────────────┼────────┼──────────────┤
│ windows │ sysmon_operational │ pipeline  │ test-events.json │ PASS   │ 253.876625ms │
╰─────────┴────────────────────┴───────────┴──────────────────┴────────┴──────────────╯
--- Test results for package: windows - END   ---
Done
```

## Related issues

- Closes https://github.com/elastic/integrations/issues/4038

### CODE REVIEW

File: packages/windows/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.19.0"
+  changes:
+    - description: Support Sysmon EventIDs - 8, 9, 19, 20, 27, 28, 255
+      type: enhancement
+      link: https://github.com/elastic/integrations/issues/4038
```
[the remark this answers is not in the document]
Comment (kcreddy) [reply packages/windows/changelog.yml:6]: Its the corresponding link to the issue. I was under impression that we should be using issue link if applicable. Let me know if not, I can change it. Thanks!

[the remark this answers is not in the document]
Comment (efd6) [reply packages/windows/changelog.yml:6]: In the [docs](https://github.com/elastic/integrations/blob/main/docs/developer_workflow_design_build_test_integration.md#remember-to-bump-up-the-version) they are pulls. I'd recommend taking the text from the issue and just including it in the commit message, like so (note that this text fixes an error in the issue):
```
- EventID 8: CreateRemoteThread[1]] | Sample Event[2]
- EventID 9: RawAccessRead[3] | Sample Event[4]
- EventID 19: WmiEventFilter activity detected[5] | Sample Event[6]]
- EventID 20: WmiEventConsumeractivity detected[7] | Sample Event[8]
- EventID 27: FileBlockExecutable[9]
- EventID 28: FileBlockShredding[10]
- EventID 255: Error[11]

[1]https://learn.microsoft.com/en-us/sysinternals/downloads/sysmon#event-id-8-createremotethread
[2]https://www.ultimatewindowssecurity.com/securitylog/encyclopedia/event.aspx?eventid=90008
[3]https://learn.microsoft.com/en-us/sysinternals/downloads/sysmon#event-id-9-rawaccessread
[4]https://www.ultimatewindowssecurity.com/securitylog/encyclopedia/event.aspx?eventid=90009
[5]https://learn.microsoft.com/en-us/sysinternals/downloads/sysmon#event-id-19-wmievent-wmieventfilter-activity-detected
[6]https://www.ultimatewindowssecurity.com/securitylog/encyclopedia/event.aspx?eventid=90019
[7]https://learn.microsoft.com/en-us/sysinternals/downloads/sysmon#event-id-20-wmievent-wmieventconsumer-activity-detected
[8]https://www.ultimatewindowssecurity.com/securitylog/encyclopedia/event.aspx?eventid=90020
[9]https://learn.microsoft.com/en-us/sysinternals/downloads/sysmon#event-id-27-fileblockexecutable
[10]https://learn.microsoft.com/en-us/sysinternals/downloads/sysmon#event-id-28-fileblockshredding
[11]https://learn.microsoft.com/en-us/sysinternals/downloads/sysmon#event-id-255-error
```

[the remark this answers is not in the document]
Comment (kcreddy) [reply packages/windows/changelog.yml:6]: Thanks @efd6

[the remark this answers is not in the document]
Comment (kcreddy) [reply packages/windows/changelog.yml]: Updated link and commit message

[the remark this answers is not in the document]
Comment (kcreddy) [reply packages/windows/changelog.yml]: > If you have publicly available origins for 27, 28 and 255, I'd add those links in as well.

Unfortunately I couldn't fine them. Had to generate them

**Review by efd6 — APPROVED** [review]

**Pull request merged**